index.html

```diff
@@ -4,17 +4,227 @@
+           
+                <div class="col-2">
+                    <form>
+                        <button class="btn btn-outline-success" type="submit" style="background-color:#CFF4EC;">Book an Appointment</button>
+                    </form>
+                </div>
+
+<!--Hamburger Icon -->
+                <div class="col-2">
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                            data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                            aria-expanded="false" aria-label="Toggle navigation">
+                            <span class="navbar-toggler-icon"></span>
+                    </button>
+                </div>
+            
+<!-- Nav Tabs -->
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0" style="justify-content: space-between;">
+                        <li class="nav-item">
+                            <a href="./index.html" class="nav-link active" aria-current="page" >Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./about.html" class="nav-link" >About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./services.html" class="nav-link">Services</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./blog.html" class="nav-link" >Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./contact.html" class="nav-link" >Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+
+        
+
+<main>
+<!-- Main Image 1 -->
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6 pt-5" style="background-color:#CFF4EC; text-align:center; align-items-center; justify-content-center"> 
+                    <h1>Online Therapy for Emotional Health</h1> 
+                    <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed
+                    pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim.
+                    Fusce est.</p>
+                </div>
+               
+                <div class="col-md-6 " style="padding: 0px;">
+                    <img class="img-fluid" src="./css/hero-background-image.jpg">
+                </div>
+            </div>
+        </div>  
+           
+<!-- Main Image 2 -->
+    <div class="container">
+        <div class="row"> 
+            <div class="col-md-10 offset-md-2" style="position: absolute; margin: 100px 0px;  background-color:#F2F2F1">
```

Comment:
10 columns wide, pushed 2 to the side: this div won't be centered. If you want to center it, only offset by 1.